Blockzilla/URLBar.swift

```diff
@@ -34,7 +34,7 @@ class URLBar: UIView {
 
     private let toolset = BrowserToolset()
     private let urlTextContainer = UIView()
-    private let urlText = URLTextField()
+    public let urlText = URLTextField()
```

Comment:
nit: Instead of making all of this public, you could create a computed property that the drag and drop can use

maybe something like: `var draggableUrlTextView: UIView { return urlText }`